frame/election-provider-multi-phase/src/benchmarking.rs

```diff
@@ -303,7 +307,8 @@ frame_benchmarking::benchmarks! {
 			..Default::default()
 		};
 
-		MultiPhase::<T>::on_initialize_open_signed().expect("should be ok to start signed phase");
+		<MultiPhase<T>>::create_snapshot().unwrap();
```

Comment:
why not have all these unwraps be a little more descriptive about what went wrong?

either with an `Err` or an `expect`

Reply:
Unless if the `expect` is going to a multi-line, comprehensive description of why something failed, writing some rudimentary `.expect(<foo failed>)` is not at all more descriptive than the standard error message of `unwrap`.

`Err` would have been nicer, but not in this case because most of these unwraps are not on dispatchables, therefore their error is not `Into<&'static str>` and needs a `map_err` or an implementation of that just for benchmarks. 

I will look into the second option, though my opinion is still that unwrap is fine in benchmark setup.

Reply:
with the help of `strum_macros`: [`e6eb8b0` (#9569)](https://github.com/paritytech/substrate/pull/9569/commits/e6eb8b043fe0d01c44b93e227241466da83bb316)